src/shared/cache/PmidCache.ts

```diff
@@ -2,34 +2,91 @@ import LazyMobXCache from "../lib/LazyMobXCache";
 import {AugmentedData} from "../lib/LazyMobXCache";
 import request from "superagent";
 
-async function fetch(pmids: number[])
+export type PmidData = {
+        [pmid:string]:PmidDatum;
+};
+
+export type PmidDatum = {
```

Comment:
Actually it might be better to name the model `PubMedRecord` and the cache `PubMedCache`. Let's merge this PR as is, we can make those changes later.